.ci/pipeline/ci.yml

```diff
@@ -36,6 +36,72 @@ jobs:
     inputs:
       artifactName: '$(platform.type) build'
       targetPath: '__release_lnx_gnu'
+# conformance
+  - script: |
+      which git
+      git --version
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
This should be placed in scripts and not in pipeline - you can take a look on daal4py or scripts in master as examples